lib/apns/core.rb

```diff
@@ -6,20 +6,24 @@ module APNS
   @host = 'gateway.sandbox.push.apple.com'
   @port = 2195
   # openssl pkcs12 -in mycert.p12 -out client-cert.pem -nodes -clcerts
-  @pem = nil # this should be the path of the pem file not the contentes
+  @pems = {} 
+  # this should be the path of the pem file not the contentes
+  # Example:
+  # { some_ios_app : "file/to/pem" }
+
   @pass = nil
 
   class << self
-    attr_accessor :host, :pem, :port, :pass
+    attr_accessor :host, :pems, :port, :pass
   end
 
-  def self.send_notification(device_token, message)
+  def self.send_notification(device_token, app, message)
```

Comment:
this will also not be backwards compatible, can we make `app` have a default value (that matches the suggestion above)

`def self.send_notification(device_token, message, app: :default)`

This kind of comment should apply to all of the other places where you modify method signatures.